content/docs/command-reference/params/index.md

```diff
@@ -257,10 +258,11 @@ stages:
       - path: users.csv
         md5: 23be4307b23dcd740763d5fc67993f11
     params:
-      INT: 5
-      BOOL: true
-      TrainConfig.EPOCHS: 70
-      TrainConfig.layers: 9
+      - params.py:
```

Comment:
I think this should be a dictionary

Reply:
@iesahin, this is a `dvc.lock` file, where `params` are in dictionary.

Reply:
Thank you. I think it should be done now.